lib/pages/home/home_header.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/gen/assets.gen.dart';
+
+class HomeHeader extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        Text(
+          // TODO
+          'MONDAY, JUNE 15',
+          style: Theme.of(context).textTheme.subtitle1!.copyWith(
+                fontSize: 13,
```

Comment:
want to have more TextTheme? 💁‍♂️

Reply:
Hmm, maybe no. There is only 1 place of this text style 🥲

Reply:
Fixed https://github.com/luongvo/flutter-survey/pull/49/commits/1abe0b2bcc9e5d9a5143c15dda51929467fdc212